src/govuk/components/select/index.js

```diff
@@ -32,9 +32,6 @@ function Select(props) {
     errorMessageComponent = <ErrorMessage id={errorId} {...errorMessage} />
   }
 
-  const selectedItem = props.items.find(item => item.selected === true)
```

Comment:
Have removed the individual `selected` values on the `items` in favour of a top level `value` prop which is what React prefers